Guides/Image_Formats.md

```diff
@@ -2,46 +2,44 @@
+The .CUE file itself is also human-readable, making it a particularly easy format to customize and alter by hand, as the .BIN file will generally be unaltered even between differing CUE standards. It also supports MANY more features compared to ISO, including multiple tracks, bad sectors, and different track types. This makes it much more suited to supporting more complicated CDs, such as games with real audio tracks present or a multitude of copy protections. However, it's very much *not* suited to DVDs or BDs, which don't contain the additional data that .CUE images are designed to store, and instead can make use of features that CUE wasn't designed to support.
 
-While this format was originally created by the program CDRWIN, it's accessibility and strong feature support meant that it was readily adopted by many other programs, for various different use cases. One of the key disadvantages of this, as already mentioned, is the relative lack of standardization of the format. But this is generally a non-issue, due to the simplicity of the format allowing users to manually change the .CUE file if needed, as long as all the needed data is present in some way. And while several features, such as sessions and subchannels, aren't typically considered supported, the format can be customized to add support for them. The issue with that then is that it takes away another of the key advantages of the format: its exceptional software support. This means that for more difficult to copy discs, other formats designed more specifically for those problems is generally used. One last limitation present in the format is that it's specifically intended for CDs. DVDs and BDs don't contain the additional data that .CUE images are designed to store, and instead can make use of features that CUE very much was *not* designed to support.
+While the CUE format was originally used by the program CDRWIN, its accessibility and strong feature support led many other programs to readily adopt it for a variety of use cases, granting it exceptional software support as well. The relative lack of standardization is a key disadvantage, but it's generally not an issue due to the simplicity of the format allowing users to manually change the .CUE file if needed, as long as all the necessary data is present in the file. And while several features, such as sessions and subchannels, aren't universal elements of the format, it can be customized to add support for them. This comes at the cost of the aforementioned wide software support, so for many CDs which are more difficult to copy, more complex formats are generally used, which were often designed with those particular issues in mind.
 
-### CloneCD Images (.CCD)
+### CloneCD Image (.CCD/.IMG)
```

Comment:
Good job catching that I missed .IMG! If you could also add "/.SUB" too please, as that's another core file used by the format.